### PR TITLE
AB#9995 tv app banners in footer

### PR DESCRIPTION
**Pull request by Jason-1**
ADO card: ☑️ [AB#9995](https://dev.azure.com/S72/fefacba9-96b6-4af2-a53d-050ed453e13a/_workitems/edit/9995)

Designs: 
- https://zpl.io/W4WpzjK Implementation has been signed off by Bex

## Description of work
Added TV app banners in the footer, below handheld banners. These will show and hide depending on configs.

## Config settings/Toggles required for the feature to work
- Content/config
- app_link_android_tv -> https://play.google.com/ (public)
- app_link_apple_tv_enabled -> true (public)
- app_link_fire_tv -> https://www.amazon.com/ (public)

Not yet documented

### Affected Clients
 - All clients who use Kibble

## Checklist
- [ ] CI tests are passing Github actions (inc. linting)
- [ ] Key areas of the feature outlined for context and testing
- [ ] If there are designs for this work are they noted here and in the ADO card 
- [ ] Design review
- [ ] Have checked this at multiple screen resolutions and range of browsers
- [ ] Moved ADO card to Dev/done, checked link to Github, tagged "Review"
- [ ] Updated changelog (if applicable)
- [ ] I promise to document any new feature toggles/configurations in the appropriate documentation

### CODE REVIEW

**Comment by l0ud0gg** [comment]
will need to document configs and update changelog when ready to merge

File: site/templates/application/footer/app-badges.jet [hunk 1]
```diff
@@ -19,6 +22,33 @@
...
+    <div class="tv-app-banner">
+      <h2 class="tv-app-badge-title">{{ i18n("tv_app_badge_title") }}</h2>
+      <div class="tv-app-badge-container">
+        {{if config("app_link_fire_tv") != ""}}
+          <div class="tv-app-badge">
+            <a class="app-badge-link" href="{{app_link_fire_tv}}" target="_blank">
+              <s72-image class="amazonfiretv" src="/images/icons/amazonfiretv.png" alt="Amazon Fire TV"></s72-image>
+            </a>
+          </div>
+        {{end}}
```
Comment (PeculiarGoat) [inline site/templates/application/footer/app-badges.jet:37]: Could do we a newline after each block for readability!

File: site/templates/application/footer/app-badges.jet [hunk 2]
```diff
@@ -19,6 +22,33 @@
         {{end}}
       </div>
     </div>
+    {{yield separator()}} {* Test these - toggle configs and test resolutions *}
+  {{end}}
+  {{if config("app_link_apple_tv_enabled") == "true" || config("app_link_android_tv") != "" || config("app_link_fire_tv") != ""}}
+    <div class="tv-app-banner">
+      <h2 class="tv-app-badge-title">{{ i18n("tv_app_badge_title") }}</h2>
+      <div class="tv-app-badge-container">
+        {{if config("app_link_fire_tv") != ""}}
```
Comment (PeculiarGoat) [inline site/templates/application/footer/app-badges.jet:31]: These can all be extracted into a separate block to reduce repetition.  (The app badge ones could use the same extracted block).

File: site/styles/_footer.scss [hunk 3]
```diff
@@ -107,9 +110,40 @@ footer {
   .app-badge-link {
     margin: 0 5px;
   }
+
   .app-badge-image {
     height: 40px;
   }
+
+  .amazonfiretv {
```
Comment (PeculiarGoat) [inline site/styles/_footer.scss:118]: None of the heights on these match what is shown in Zeplin.

And none of them need to have padding or margins.

File: site/styles/_footer.scss [hunk 4]
```diff
@@ -107,9 +110,40 @@ footer {
...
+    padding-left: 23px;
+  }
+
+  .appletv {
+    height: 44px;
+    margin-top: -3px;
+    padding-left: 23px;
+  }
+
+  .tv-app-badge-container {
```
Comment (PeculiarGoat) [inline site/styles/_footer.scss:136]: I don't think this does anything?

**Review by PeculiarGoat — APPROVED** [review]
Nice work!  👍

**Pull request merged**